README.md

```diff
@@ -1,3 +1,10 @@
 # Google Classroom AutoGrader | step25-2020
 
-This repo contains the source code for the STEP capstone project of Ben, Mahima, and Sampson.
\ No newline at end of file
+This repo contains the source code for the STEP capstone project of Ben, Mahima, and Sampson.
+
+One big change we’re seeing right now (2020) is schools all over the country being moved online this Fall due to the COVID-19 Pandemic. Teachers will have to shift to giving instruction online. However, teachers are already overworked, oftentimes working 51-53 hours weeks on average, with 25-35% of their time being spent grading and planning. We can even expect this amount to increase as teachers have to grade online submissions. Not only may this make grading for teachers more time consuming, but difficult to use as many teachers still use worksheets as assignments in primary and secondary education. Grading digital submissions may also prove to be an increased challenge as more than 50% of teachers are aged 40+ years, meaning the average user (teacher) may also be technologically challenged, and struggle with navigating the web.
```

Comment:
I think this should not imply that older people are technologically challenged and instead focus on the shift to online instruction

Reply:
okay